### PR TITLE
Timedistance Updates

### PR DESCRIPTION
**Pull request by kdiluca**
This PR goes along with the Thor PR for timedistance.  This includes a new matrix_action.cc.  There may be a few more small updates.

### CODE REVIEW

File: src/loki/route_action.cc [hunk 1]
```diff
@@ -36,29 +36,6 @@ namespace {
             break;
```
Comment (kevinkreiser) [inline src/loki/route_action.cc]: remove this function entirely its not used by route

File: src/loki/service.cc [hunk 2]
```diff
@@ -193,7 +198,6 @@ namespace valhalla {
         throw std::runtime_error("Insufficient number of locations provided");
       LOG_INFO("location_count::" + std::to_string(request_locations->size()));
 
-      //using the costing we can determine what type of edge filtering to use
```
Comment (kevinkreiser) [inline src/loki/service.cc]: why would you remove this comment?

File: src/loki/route_action.cc (hunk 1 above)
Comment (kdiluca) [inline src/loki/route_action.cc]: moved it to another part of the code unintentionally

**Comment by kdiluca** [comment]
Just realized that I'm not pulling the correct max_area and max_locations from the config.  Fixing now.

**Pull request merged**